src/main/webapp/app/account/login-form/login-form.component.ts

```diff
@@ -6,8 +6,8 @@ import AccountService from '@/account/account.service';
   watch: {
     $route() {
       this.$root.$emit('bv::hide::modal', 'login-page');
-    },
```

Comment:
why are you removing this ? is it manually or automatically ?
Just try to run `npm run prettier:format` and it will add it back

Reply:
Yes it was removed automatically, thank you @pascalgrimaud